ipie/addons/ephqmc/hamiltonians/generic.py

```diff
@@ -0,0 +1,11 @@
+import numpy
+from ipie.hamiltonians.generic_base import GenericBase
+from ipie.utils.backend import arraylib as xp
+
+
+class GenericElphBase(GenericBase):
```

Comment:
What is this base class for? Is it really used somewhere?

Reply:
not yet really, I'll remove it for now 👍🏼